src/test/StdCheats.t.sol

```diff
@@ -76,6 +76,60 @@ contract StdCheatsTest is Test {
         assertEq(barToken.totalSupply(), 10000e18);
     }
 
+    function testBound() public {
+        assertEq(bound(5, 0, 4), 0);
+        assertEq(bound(0, 69, 69), 69);
+        assertEq(bound(0, 68, 69), 68);
+        assertEq(bound(10, 150, 190), 174);
+        assertEq(bound(300, 2800, 3200), 3107);
+        assertEq(bound(9999, 1337, 6666), 4669);
+        
+    }
+
+    function testRevertBoundMinBiggerThanMax() public {
+        vm.expectRevert(bytes("MAX_LESS_THAN_MIN"));
+        bound(5, 100, 10);
+    }
+
+    function testRelApproxEqBothZeroesPasses() public {
+        assertRelApproxEq(0, 0, 1e18);
+        assertRelApproxEq(0, 0, 0);
+    }
+
+    function testBound(
+        uint256 num,
+        uint256 min,
+        uint256 max
+    ) public {
+        if (min > max) (min, max) = (max, min);
+
+        uint256 bounded = bound(num, min, max);
+
+        assertGe(bounded, min);
+        assertLe(bounded, max);
+    }
+
+    function testBound_Uint256Max() public {
```

Comment:
Let's keep the code style consistent; use `PascalCase` here.
```suggestion
    function testBoundUint256Max() public {
```

Reply:
Updated function naming style.